codegen-luau/runtime/runtime.lua

```diff
@@ -529,6 +529,24 @@ do
 	end
 
 	local store_i32 = store.i32
+	local store_i32_n8 = store.i32_n8
+	local store_i32_n16 = store.i32_n16
+
+	function store.i64_n8(memory, addr, value)
+		local data_1, data_2 = num_into_u32(value)
+
+		store_i32_n8(memory, addr, data_1)
```

Comment:
We're storing a `i64` so we also need to zero out the higher address of the integer.

Reply:
Forgive me, but I don't really understand what you're referring to. Only the lower 32 bits of the number are used, the upper 32 bits are discarded, and the 32 bit integer is then passed to `store_i32_n8`. Wouldn't this mean the implementation of `store_i32_n8` is incorrect?

Reply:
My bad, looks like I misread my own code.